presto/presto.go

```diff
@@ -1307,30 +1302,18 @@ func newTypeConverter(typeName string, signature typeSignature) (*typeConverter,
 	}
 	switch signature.RawType {
 	case "char", "varchar":
-		if len(signature.Arguments) > 0 {
-			if signature.Arguments[0].Kind != KIND_LONG {
-				return nil, ErrInvalidResponseType
-			}
+		if len(signature.Arguments) > 0 && signature.Arguments[0].Kind == KIND_LONG {
```

Comment:
Updated all of these to be more forgiving, if the type arguments aren't present in the way we'd expect. We don't really care that much if the column's size/precision/scale is available or not.